pkg/resources/tag_masking_policy_association.go

```diff
@@ -84,7 +85,8 @@ func TagMaskingPolicyAssociation() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			StateContext: schema.ImportStatePassthroughContext,
 		},
-		Description: "Attach a masking policy to a tag. Requires a current warehouse to be set. Either with SNOWFLAKE_WAREHOUSE env variable or in current session. If no warehouse is provided, a temporary warehouse will be created.",
+		Description:        "Attach a masking policy to a tag. Requires a current warehouse to be set. Either with SNOWFLAKE_WAREHOUSE env variable or in current session. If no warehouse is provided, a temporary warehouse will be created.",
+		DeprecationMessage: deprecatedResourceDescription(string(providerresources.TagAssociation)),
```

Comment:
in the migration guide you listed snowflake_tag and not tag_association (I think it should be changed here)

Reply:
Yup, I meant `tag`. Fixed.